tests/integration-tests/tests/update/test_update.py

```diff
@@ -439,3 +441,144 @@ def get_batch_spot_bid_percentage(stack_name, region):
         .get("computeResources")
         .get("bidPercentage")
     )
+
+
+@pytest.mark.usefixtures("instance")
+def test_queue_parameters_update(
+    region, os, pcluster_config_reader, ami_copy, clusters_factory, scheduler_commands_factory, request
+):
+    """Test update cluster with drain strategy."""
+    # Create cluster with initial configuration
+    ec2 = boto3.client("ec2", region)
+    pcluster_ami_id = retrieve_latest_ami(region, os, ami_type="pcluster", request=request)
+    init_config_file = pcluster_config_reader(global_custom_ami=pcluster_ami_id)
+    cluster = clusters_factory(init_config_file)
+    remote_command_executor = RemoteCommandExecutor(cluster)
+    scheduler_commands = scheduler_commands_factory(remote_command_executor)
+    _check_queue_ami(cluster, ec2, pcluster_ami_id, "queue1")
+    _check_queue_ami(cluster, ec2, pcluster_ami_id, "queue2")
+
+    _test_queue_strategy_without_running_job(
+        pcluster_config_reader, pcluster_ami_id, cluster, region, os, remote_command_executor, scheduler_commands
+    )
+
+    queue1_job_id = scheduler_commands.submit_command_and_assert_job_accepted(
+        submit_command_args={
+            "command": "sleep 3000",
+            "nodes": -1,
```

Comment:
Q. why nodes -1?

Reply:
It means not adding the node option,
 in our code:
```
    def _submit_batch_job(
        self,
        job_submit_command,
        nodes=0,
        slots=None,
        host=None,
        after_ok=None,
        partition=None,
        constraint=None,
        other_options=None,
        additional_files=None,
        raise_on_error=True,
    ):
        submission_command = "sbatch"
        if host:
            submission_command += " --nodelist={0}".format(host)
        if slots:
            submission_command += " -n {0}".format(slots)
        if nodes > 0:
            submission_command += " -N {0}".format(nodes)
```